src/Microsoft.IdentityModel.Xml/KeyInfo.cs

```diff
@@ -115,7 +115,7 @@ public override int GetHashCode()
         /// <summary>
         /// Returns true if the KeyInfo object can be matched with the specified SecurityKey, returns false otherwise.
         /// </summary>
-        internal bool MatchesKey(SecurityKey key)
```

Comment:
Moving this to protected internal, we need to make the doc string on this method more useful

Reply:
@keegan-caruso  how do we do that?